CMakeLists.txt

```diff
@@ -53,26 +76,26 @@ target_compile_options(${PROJECT_NAME} PUBLIC "$<$<COMPILE_LANG_AND_ID:CXX,MSVC>
 # Link dependencies
 target_link_libraries(${PROJECT_NAME} PRIVATE fmt::fmt)
 
-target_include_directories(
-  ${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
-                         $<INSTALL_INTERFACE:include/${PROJECT_NAME}-${PROJECT_VERSION}>
-)
+set(GREETER_VERSION \"${PROJECT_VERSION}\")
+string(TOUPPER ${PROJECT_NAME} UPPERCASE_PROJECT_NAME)
+configure_file(${PACKAGE_PROJECT_ROOT_PATH}/version.h.in ${VERSION_HEADER_LOCATION} @ONLY)
+add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
+
+if(CMAKE_SKIP_INSTALL_RULES)
+  return()
+endif()
```

Comment:
@TheLartians Note: this is a conceptual problem wenn installation shall be disabled

The library `ALIAS` and the generated headers are missing!